api/db/populate-comments.js

```diff
@@ -35,16 +36,14 @@ const prerequisites = (eventsCount, attendantsCount) => {
 
 const getParentComment = (comments) => {
   const parentComments = comments.filter(
-    (comment) => comment.parentComment === undefined
+    (comment) =>
+      comment.parentComment === undefined || comment.parentComment === null
```

Comment:
Can't this be simply written as `(comment) => !comment.parentComment`? I see now where this error came from though, so I'll leave a comment about it right below.